docs/installation/installation-framework.md

```diff
@@ -85,3 +97,52 @@ If you are on a machine without an attached display (e.g. a server):
 conda install habitat-sim=0.1.5 headless -c conda-forge -c aihabitat --name allenact
 ```
 
+#### Installing a Conda environment
```

Comment:
I really like this more thorough explanation here. Didn't catch anything I'd change (I think it's good to move it here, where it is more contextualized).